content/en/real_user_monitoring/retention_analysis/index.md

```diff
@@ -0,0 +1,79 @@
+
+{{< img src="real_user_monitoring/retention_analysis/example-retention-analysis-graph.png" alt="Example Retention Analysis graph" style="width:100%;" >}}
+
+## Build a graph
+
+To build a retention graph, navigate to **[UX Monitoring > Product Analytics > Retention Analysis][1]**, then follow the steps below.
+
+### 1. Define the initial event
+1. Select the view or action to act as the initial event for defining a group of users.
+2. Optionally, add filters such as the device used or the country of origin.
+    - If your initial event is a view, you can add any [view facets][2] or context facets.
+    - If your initial event is an action, you can add any [action facets][3] or context facets.
+
+### 2. Optionally, define the return event
+The return event defaults to a repeat of the original event. To use a different return event: 
+
+1. Change **Repeated the original event** to **Experienced a different event**.
+2. Select the view or action to act as the return event.
+3. Optionally, add any desired filter criteria.
+
+## Analyze the graph
+For insights on user retention week over week, read each row of the graph horizontally from left to right. 
+
+You can click on an individual diagram cell to view a list of users, and export the list as a CSV:
+
+{{< img src="real_user_monitoring/retention_analysis/retention-analysis-graph-details-panel.png" alt="Details panel for a diagram cell" style="width:90%;" >}}
+
+The graph displays slightly different information depending on whether the initial and return events match.
+
+### Matching events
+If the events match:
+- **Week 0** is always 100%, since it represents all of the users who completed the initial event.
+- The other cells compare the viewers in a given week to **Week 0**, displaying the percentage of the cohort who completed the event in that week.
+
+{{< img src="real_user_monitoring/retention_analysis/matching-events-retention-graph.png" alt="Retention graph for matching events" style="width:90%;" >}}
+
+Reading the **Dec 04 2023** row of the above graph from left to right:
+- 94% of the people who completed the event in **Week 0** came back and completed it again in **Week 1**.
+- 92% of the people who completed the event in **Week 0** came back and completed it again in **Week 2**.
+
+### Differing events
+If the events differ:
+- **Week 0** represents users who completed both the initial and return events.
+- After **Week 0**, each cell displays the percentage of the **Users** column who completed the return event in that week.
+
+{{< img src="real_user_monitoring/retention_analysis/differing-events-retention-graph.png" alt="Retention graph for differing events" style="width:90%;" >}}
+
+Reading the **Dec 04 2023** row of the above graph from left to right:
+- 144 users completed the initial event.
+- In **Week 0**, 94% of those 144 users completed the return event.
+- In **Week 1**, 92% of the 144 users completed the return event.
+
+## Limitations
```

Comment:
do you think this name may be harsh to write? i wonder if there a better phrasing for this